doc/nrf-bm/api/api.rst

```diff
@@ -0,0 +1,149 @@
+.. nrf_bm_api:
+
+API documentation
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. _api_libs:
+
+Libraries
+*********
+
+Bluetooth LE Connection Parameter library
+=========================================
+
+.. doxygengroup:: ble_conn_params
+
+Bluetooth LE MCUmgr Service library
+===================================
+
+.. doxygengroup:: ble_mcumgr
+
+Bluetooth LE Nordic UART Service library
+========================================
+
+.. doxygengroup:: ble_nus
+
+Bluetooth LE advertising library
+================================
+
+.. doxygengroup:: ble_adv
+
+Advertising and Scan Response Data Encoder
+------------------------------------------
+
+.. doxygengroup:: ble_sdk_lib_advdata
+
+Bare Metal Zephyr Memory Storage (ZMS)
+======================================
+
+.. doxygengroup:: bm_zms
+
+nRF Connect SDK Bare Metal Buttons library
```

Comment:
ZMS doesn't have this part in the title. Also, I think it's a bit too extra in my opinion and we can remove it for all entries here. `Bare Metal` should be sufficient.
```suggestion
Bare Metal Buttons library
```